sentiment_api.py

```diff
@@ -13,96 +13,110 @@
 load_dotenv()
 
 # MongoDB Connection
-MONGO_URI = os.getenv('MONGO_URL')
-DB_NAME = os.getenv('DB_NAME')
-COLLECTION_NAME = os.getenv('COLLECTION_NAME')
+MONGO_URI = os.getenv("MONGO_URL")
+DB_NAME = os.getenv("DB_NAME")
+COLLECTION_NAME = os.getenv("COLLECTION_NAME")
 
 client = MongoClient(MONGO_URI)
 db = client[DB_NAME]
 collection = db[COLLECTION_NAME]
 
-# Create text index on headline and summary fields if it doesn't exist
-if "headline_summary_text" not in collection.index_information():
-    collection.create_index([("headline", "text"), ("summary", "text")], 
-                           name="headline_summary_text")
+# # Create text index on headline and summary fields if it doesn't exist
```

Comment:
This line was doing full-text search. ask gpt about it. essentially it makes keyword search easier when doing with 2 or more db columns.

Commented it out to opt for regex search so we can search both headline and summary. in line 67-71.